[inductor] Update triton pin

peterbell10 opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* #102737
* #102486
* __->__ #102736

There is some bug in triton's handling of `tl.reduce` that breaks the variance PR, but is fixed on the latest triton master.

cc @voznesenskym @penguinwu @EikanWang @jgong5 @Guobing-Chen @XiaobingSuper @zhuhaozhe @blzheng @Xia-Weiwen @wenzhe-nrv @jiayisunx @ipiszy @ngimel @yf225 @aakhundov

peterbell10 committed:
[inductor] Update triton pin

[ghstack-poisoned]

peterbell10 committed:
Limit threads used for triton build on "[inductor] Update triton pin"


There is some bug in triton's handling of `tl.reduce` that breaks the variance PR, but is fixed on the latest triton master.

[ghstack-poisoned]

peterbell10 committed:
Update on "[inductor] Update triton pin"


There is some bug in triton's handling of `tl.reduce` that breaks the variance PR, but is fixed on the latest triton master.

[ghstack-poisoned]

peterbell10 committed:
Update on "[inductor] Update triton pin"


There is some bug in triton's handling of `tl.reduce` that breaks the variance PR, but is fixed on the latest triton master.

[ghstack-poisoned]

peterbell10 committed:
Update on "[inductor] Update triton pin"


There is some bug in triton's handling of `tl.reduce` that breaks the variance PR, but is fixed on the latest triton master.

[ghstack-poisoned]

peterbell10 committed:
Update on "[inductor] Update triton pin"


There is some bug in triton's handling of `tl.reduce` that breaks the variance PR, but is fixed on the latest triton master.

[ghstack-poisoned]

peterbell10 committed:
Update on "[inductor] Update triton pin"


There is some bug in triton's handling of `tl.reduce` that breaks the variance PR, but is fixed on the latest triton master.

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng Xia-Weiwen wenzhe-nrv jiayisunx

[ghstack-poisoned]

peterbell10 committed:
Update on "[inductor] Update triton pin"


There is some bug in triton's handling of `tl.reduce` that breaks the variance PR, but is fixed on the latest triton master.

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng Xia-Weiwen wenzhe-nrv jiayisunx

[ghstack-poisoned]

ezyang commented:
Any chance we can start landing this? cc @bertmaher

yuguo68 commented:
> @yuguo68 Bert tells me you're oncall for the internal update

update diff stack is ready, doing more testing.

peterbell10 committed:
Rebase on "[inductor] Update triton pin"


There is some bug in triton's handling of `tl.reduce` that breaks the variance PR, but is fixed on the latest triton master.

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng Xia-Weiwen wenzhe-nrv jiayisunx

[ghstack-poisoned]

peterbell10 committed:
Update on "[inductor] Update triton pin"


There is some bug in triton's handling of `tl.reduce` that breaks the variance PR, but is fixed on the latest triton master.

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng Xia-Weiwen wenzhe-nrv jiayisunx

[ghstack-poisoned]

peterbell10 committed:
Update on "[inductor] Update triton pin"


There is some bug in triton's handling of `tl.reduce` that breaks the variance PR, but is fixed on the latest triton master.

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng Xia-Weiwen wenzhe-nrv jiayisunx

[ghstack-poisoned]

peterbell10 committed:
Update on "[inductor] Update triton pin"


There is some bug in triton's handling of `tl.reduce` that breaks the variance PR, but is fixed on the latest triton master.

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng Xia-Weiwen wenzhe-nrv jiayisunx

[ghstack-poisoned]

peterbell10 committed:
Update on "[inductor] Update triton pin"


There is some bug in triton's handling of `tl.reduce` that breaks the variance PR, but is fixed on the latest triton master.

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng Xia-Weiwen wenzhe-nrv jiayisunx

[ghstack-poisoned]
